test/parallel/test-http-set-timeout-server.js

```diff
@@ -213,3 +213,16 @@ test(function fastTimeout(cb) {
     });
   }));
 });
+
+test(function serverTimeout(cb) {
+  const server = http.createServer();
+  server.listen(common.mustCall(() => {
+    const timeoutValue = 50;
+    const s = server.setTimeout(timeoutValue);
+    assert.equal(s.timeout, timeoutValue);
+    assert.ok(s instanceof http.Server);
+    http.get({
+      port: server.address().port
+    }).on('error', common.mustCall());
+  }));
+});
```

Comment:
Missing newline at end of file, probably will fail ESLint because of this?

Reply:
Oops! :stuck_out_tongue: